sysutils/libvirt/Portfile

```diff
@@ -29,13 +30,14 @@ homepage            https://libvirt.org
 master_sites        ${homepage}/sources/
 use_xz              yes
 
-depends_build       port:autoconf \
-                    port:automake \
+set python_version  39
+set python_branch   3.9
```

Comment:
To simplify changing python going forward, use this to set the `python_branch`. Try this untested code based on similar code from the python 1.0 portgroup. 


`set python_branch   {[string index ${python_version} 0].[string range ${python_version} 1 end]}
`

Reply:
That looks fine too.